.cirrus.yml

```diff
@@ -464,7 +482,7 @@ special_testing_rootless_task:
 
 
 special_testing_in_podman_task:
-
+    alias: "special_testing_in_podman"
```

Comment:
FWIW, I don't believe the under-scores are important,  can you use actual spaces here?

Suggestion:  Could these test names be shortened even more to "p-in-p testing"?  Even "In Podman" might be good enough for this audience.

Reply:
I would **LOVE** to shorten even more! I just have no idea what these test names even are supposed to mean. "Special" has no meaning to me. I'm happy to change it to "in podman" (did you really want mixed case?)

Reply:
Well! I would **LOVE**  to explain WTF I was thinking when I crafted such a obscure name :smile: 

The primary and most important work done by Cirrus-CI on this repo. is the big "Integration + System testing matrix".  In other words, if any of them fail, nearly all other test results can be ignored.

So "Special" in this case simply denotes automation which is operating on something other than the primary concern.  A different (named) mode of operation (they're all documented) if you will.

Note: In a perfect world, we would have these "special" tasks depend on the primary integration/system tasks.  However, that would badly hurt overall runtime, so we do the "special" stuff in parallel.

hope that helps.

Reply:
Other than the variable names, nobody depends on the term "special" AFAIK.  Some other ideas:  Auxiliary, aux, other, non-standard, different, alternative, alt.

Though I question if the distinction is needed at all, developers could probably tell what it is just from the value of `$SPECIALMODE`.